apps/scripts/fib.c

```diff
@@ -0,0 +1,15 @@
+int fib(int n)
```

Comment:
There is not much sense in negative values here, so n should be unsigned int

Reply:
i accept